packages/strapi-admin/admin/src/themes/colors.js

```diff
@@ -14,6 +14,10 @@ const colors = {
   'gray-light': '#636c72',
   'gray-lighter': '#eceeef',
   'gray-lightest': '#f7f7f9',
+  brightGrey: '#f0f3f8',
+  lightGrey: '#fafafa',
+  grey: '#9ea7b8',
+  greyDark: '#292b2c',
```

Comment:
Should we use only KebabCase or CamelCase, or do we want to enable both?

Reply:
It's a good point really! However, I didn't felt like going over the files of the `strapi-admin` to make the updates as it doesn't concern this PR plus since I am colorblind I am not even sure that grey corresponds to hexa code I set...

Reply:
Agree!